backend/startup.sh

```diff
@@ -7,4 +7,4 @@ set -e
 # When running inside the cluster, the k8s service host should not use the proxy
 export no_proxy=$no_proxy,$KUBERNETES_SERVICE_HOST,$no_proxy_additional,svc.cluster.local
 export NO_PROXY=$NO_PROXY,$KUBERNETES_SERVICE_HOST,$no_proxy_additional,svc.cluster.local
-uvicorn capellacollab.__main__:app --host 0.0.0.0 --forwarded-allow-ips '*'
+/opt/backend/.venv/bin/uvicorn capellacollab.__main__:app --host 0.0.0.0 --forwarded-allow-ips '*'
```

Comment:
Is there a reason this is not `uv run uvicorn`?

Reply:
I tried that, but it didn't work. `uv` tried to rebuild the backend (I suppose due to the changed user), which is not what we want.